src/components/layout/footer-route-options.tsx

```diff
@@ -114,23 +116,23 @@ class FooterRouteOptions extends React.Component<Props, State> {
   /**
    * Renders routing mode and transportation switches
    */
-  private renderRoutingSwitchesSection = () => {
+  private renderRoutingSwitchesSection = () => { 
```

Comment:
```suggestion
  private renderRoutingSwitchesSection = () => {
```
Extra space to be removed